### DISCUSSION TITLE
Vercel is acting weird. I followed every videos and website I can find.

### DISCUSSION THREAD

**Discussion by joshuangdev** [vercel/community#2779] [Help] [answered]
### Summary
Hello I'm new to Vercel. I would love your help to solve these issues.

Test1:
Vercel read my index.py as a framework. But `return render_templete` doesn't work.
https://github.com/apsw171118/joshuang.profile-origamikingdom-v1
Here's my file github. Should be public. 

Test2:
Same issue. I mean it says serverless failed I dunno what happened. I'm deploying online on github both of them. git clone take 4 minutes to update.
https://github.com/apsw171118/origaimkingdom-website

# I'm starting to doubt my flask skills. My brain is failing me. Please help.

### Example

Test1: https://joshuang.origamikingdom.de Test2: [origaimkingdom-website.vercel.app](https://origaimkingdom-website.vercel.app/) 

### Steps to Reproduce

Fork my github repo. Try it,  I bet it fails.

**Comment by amyegan** [comment]
Hi @apsw171118! I think you may need to alter the structure of your project to get Flask working in Serverless Functions on Vercel. When debugging, it's usually easiest for me if I can compare with some working example code. There are a few starter projects that you can use for code comparison. 

- Flask Hello World
  - template: https://vercel.com/templates/python/flask-hello-world
  - repo: https://github.com/vercel/examples/tree/main/python/flask
- Python Hello World
  - template: https://vercel.com/templates/python/python-hello-world
  - repo: https://github.com/vercel/examples/tree/main/python/hello-world
- Next.js Flask Starter
  - template: https://vercel.com/templates/next.js/nextjs-flask-starter
  - repo: https://github.com/vercel/examples/tree/main/python/nextjs-flask
- Flask Alt Text Generator
  - template: https://vercel.com/templates/python/flask-alt-text-generator
  - repo: https://github.com/Nutlope/alt-text-generator

**Comment by joshuangdev** [comment, marked as answer]
**Weeeeeeeeeeeeeee**
This issue has been fixed!

Drop me an email on ng_joshua@hotmail.com or discord on kittydevs. I'm most likely to respond on discord quicker tho..
I'm just not bothered to type it out am sorry..

**Reply by tesaabi** [reply]
how did u solve the 2nd problem? because i got the same one
